File: skills/sn-setup/templates/lang/ts/Makefile.ts
# Appended to the project Makefile by sn-init for --lang=ts.
.PHONY: install fmt lint test mcp-server agent-run client-run build

install:
	npm install

fmt:
	npx prettier --write "src/**/*.ts" "mcp_server/**/*.ts" "tests/**/*.ts" ".claude/hooks/*.ts"

lint:
	npx tsc --noEmit

test:
	npm test

build:
	npm run build

agent-run:
	npm run agent

client-run:
	npm run client

orchestrator-run:
	npm run orchestrator -- $(SPRINT)

mcp-server:
	npm run mcp-server

File: skills/sn-setup/templates/lang/go/Makefile.go
# Appended to the project Makefile by sn-init for --lang=go.
.PHONY: install fmt lint test mcp-server agent-run

install:
	go mod tidy

fmt:
	go fmt ./...

lint:
	go vet ./...

test:
	go test ./...

agent-run:
	go run ./src

client-run:
	go run ./src

orchestrator-run:
	go run ./src/cmd/orchestrator $(SPRINT)

mcp-server:
	go run ./mcp_server
